priority-queue/PriorityQueueConsumerLow/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "appInsights1": {
+      "type": "appInsights",
+      "connectionId": "APPINSIGHTS_CONNECTIONSTRING"
```

Comment:
This line (`connectionId`) exists in Low but doesn't exist in High. Any reason they have to be different?

Reply:
Fixed. See response to comment above

Reply:
Thanks for syncing them up.